service/src/main/java/bio/terra/workspace/service/grant/flight/RevokeStep.java

```diff
@@ -288,6 +296,52 @@ private void revokeResourceNotebook(
     }
   }
 
+  private void revokeResourceGceInstance(
```

Comment:
minor: this looks very similar to the one for Notebooks. Would a common function help? something on the lines of 

revokeResourceGrant(SetIamPolicy, grantData, ResourceType) {}